content/1.documentation/2.clients/1.cli.md

```diff
@@ -49,16 +49,40 @@ hopp test -e environment.json kitchen-sink-hoppscotch-collection.json
 
 Hoppscotch allows templates in several places. For example, you could specify your endpoint URL as `<<baseurl>>/post` and specify baseurl as [`https://echo.hoppscotch.io`](https://echo.hoppscotch.io) in an environment file.
 
-The expected format of an environment file is:
+Hoppscotch CLI supports environment files in two specific formats:
+
+### 1. Multiple Environment Variables Export Format
```

Comment:
It's better to name this to Single Environment Entry Export Format. The current title may give the idea that we are support bulk environment export which the CLI doesn't support.